test/src/test/java/org/corfudb/runtime/view/QuorumReplicationStreamViewTest.java

```diff
@@ -0,0 +1,28 @@
+package org.corfudb.runtime.view;
+
+import org.junit.Before;
+import org.junit.Ignore;
+
+import java.util.Collections;
+
+/**
+ * Created by kspirov
+ */
+@Ignore
+public class QuorumReplicationStreamViewTest extends StreamViewTest {
+
+    @Before
+    @Override
+    public void setRuntime() throws Exception {
+        r = getDefaultRuntime().connect();
+        // First commit a layout that uses Replex
+        Layout newLayout = r.layout.get();
+        newLayout.getSegment(0L).setReplicationMode(Layout.ReplicationMode.CHAIN_REPLICATION);
+        newLayout.getSegment(0L).setReplexes(Collections.singletonList(
```

Comment:
Copy/paste error. :) Will send you on Monday a  bit corrected version, along with the quorum replication draft.